corehq/apps/commtrack/static/commtrack/js/sms.js

```diff
@@ -184,15 +184,21 @@ hqDefine('commtrack/js/sms.js', function () {
             };
         };
     }
-    return {
-        initCommtrackSettingsView: function ($element, settings, other_sms_codes) {
-            var model = new CommtrackSettingsViewModel(other_sms_codes);
-            $element.submit(function () {
-                return model.presubmit();
-            });
 
-            model.load(settings);
-            $element.koApplyBindings(model);
-        }
-    };
+    function initCommtrackSettingsView($element, settings, other_sms_codes) {
+        var model = new CommtrackSettingsViewModel(other_sms_codes);
+        $element.submit(function () {
+            return model.presubmit();
+        });
+
+        model.load(settings);
+        $element.koApplyBindings(model);
+    }
+
+    $(function () {
```

Comment:
Eventually everything should be in an `hqDefine`, but I haven't been making any effort to make progress on that front. But, as long as I'm touching all of these files, it does make sense to add in `hqDefine` blocks in the places where it's easy to do so (basically the same approach I've been taking to linting). Does that work? I can take a quick pass at the PRs I've done thus far.